integration_tests/e2e/about.cy.ts

```diff
@@ -158,3 +151,29 @@ describe('Rendering', () => {
     cy.contains('.govuk-accordion__section', 'Health and wellbeing').find('.govuk-body').should('have.length', 2)
   })
 })
+
+describe('Rendering READ_ONLY', () => {
+  beforeEach(() => {
+    cy.createSentencePlan().then(planDetails => {
+      cy.wrap(planDetails).as('plan')
+      cy.openSentencePlan(planDetails.oasysAssessmentPk, AccessMode.READ_ONLY)
+      cy.get('.moj-primary-navigation__container').contains('a', 'About').click()
+    })
+  })
+
+  it('Should check the page rendered correctly with no Create Goal button', () => {
+    cy.url().should('include', '/about')
+    cy.get('h1').should('include.text', 'About')
+    cy.get('h2').eq(0).contains('Sentence information')
+    cy.get('h2').eq(1).contains('High-scoring areas from the assessment')
+    cy.get('[role="button"]').should('have.length', 1)
```

Comment:
No change needed but, just interested in your opinion;
* This will break the test if we ever added a clickable button elsewhere into the About Person page that is useable in the READ_ONLY view.
* Counterpoint, if the page has been updated, the test should likely be updated anyway

Reply:
Yes I think I would want this test to break if another button was added because as you say, the selector for this particular button will need to be updated.